spec/requests/content_blocks_spec.rb

```diff
@@ -15,6 +15,8 @@ def compare_content_blocks(object, content_block)
   end
 
   context '#index' do
+    before(:each) { Rails.cache.clear }
```

Comment:
the problem was, that other specs already created a cache. i believe it would have been enough to clear the cache before all:

```rb
    before(:all) { Rails.cache.clear }
```